modules/wyrt/src/whiley/lang/Int.whiley

```diff
@@ -188,7 +188,7 @@ public function parse(string input) -> int|null:
     while i < |input|:
         char c = input[i]
         r = r * 10
-        if !ASCII.isDigit(c):
```

Comment:
This stopped the runtime library from compiling. It was a bit hard to track down because the error didn't say the file or line number that was causing the problem.

```

compile-wyrt:

BUILD FAILED
/p/whiley/compiler/build.xml:24: The following error occurred while executing this line:
/p/whiley/compiler/modules/wyrt/build.xml:6: wycc.lang.SyntaxError: unknown variable
  at wycc.lang.SyntaxError.syntaxError(SyntaxError.java:321)
  at wyc.lang.WhileyFile.syntaxError(WhileyFile.java:618)
  at wyc.builder.FlowTypeChecker.propagate(FlowTypeChecker.java:1822)
  at wyc.builder.FlowTypeChecker.propagate(FlowTypeChecker.java:1373)
  at wyc.builder.FlowTypeChecker.propagateCondition(FlowTypeChecker.java:1027)
  at wyc.builder.FlowTypeChecker.propagateCondition(FlowTypeChecker.java:1060)
```